src/main/java/com/github/fishio/PlayerFish.java

```diff
@@ -271,7 +263,7 @@ public void setRightPressed(boolean rightPressed) {
 	 * @return The rate at which the PlayerFish grows.
 	 */
 	public double getGrowthSpeed() {
-		return GROWTH_SPEED;
+		return settings.getInteger("GROWTH_SPEED");
```

Comment:
I asked Luka, he said an int was fine because the values are big (default = 500).

<organisation, review